includes/cli/class-setup.php

```diff
@@ -139,7 +139,7 @@ private function initial_content() {
 	 */
 	private function campaigns_config() {
 		global $table_prefix;
-		$filename = WP_CONTENT_DIR . '/newspack-popups.config.php';
+		$filename = WP_CONTENT_DIR . '/newspack-popups-config.php';
```

Comment:
In principle, [`Newspack_Popups::LIGHTWEIGHT_API_CONFIG_FILE_PATH`](https://github.com/Automattic/newspack-popups/blob/8aa5ac219ed3c6a6fab180af47a8b6df2f1813b9/includes/class-newspack-popups.php#L22) should be used.

Reply:
The popups plugin is not necessarily (and not by default) active when this command runs